app/views/users/admin/queue.scala.html

```diff
@@ -85,11 +89,11 @@ <h3>@messages("queue.review.none")</h3>
                                     @if(currentReview.isDefined) {
                                         @users.get(currentReview.get.userId).get.name
                                         <br>
-                                        <span data-ago="@currentReview.get.createdAt" data-title="started "></span>
+                                        <span data-ago="@currentReview.get.createdAt.getOrElse(Timestamp.from(Instant.now())).getTime" data-title="started "></span>
```

Comment:
This prevents a NPE from being thrown, but it doesn't fix the timezone being off.

Reply:
I think it does fix it because now its getting the time since Epoch instead of like "2018-02-25 22:31:12.200" but I could be mistaken

Reply:
It does that only if `createdAt` is null, which should preferably never be null.

Reply:
That's wrong @phase he added .getTime which is called on createdAt If present or otherwise on Instant.now()

Reply:
Ah, I misread.